### PR TITLE
Feature/rnp 100 mobile auth confirmation

### PR DESCRIPTION
**Pull request by MichielVrins**

### CODE REVIEW

**Review by PiotrKlis — APPROVED** [review]
Kotlin files are ok 👍

File: ios/Handlers/MobileAuthHandler.swift [hunk 1]
```diff
@@ -49,14 +50,15 @@ extension MobileAuthHandler : MobileAuthConnectorToHandlerProtocol {
...
 
-    func handleMobileAuthConfirmation(accepted: Bool) {
+    func handleMobileAuthConfirmation(accepted: Bool) -> Bool {
         if authenticatorType == .fingerprint {
             //@todo
         } else if authenticatorType == .confirmation {
-            handleConfirmationMobileAuth(accepted)
+            return handleConfirmationMobileAuth(accepted)
         } else if authenticatorType == .pin {
             //@todo
```
Comment (kfa-onewelcome) [inline ios/Handlers/MobileAuthHandler.swift:59]: are these `todos` still needed ?

File: ios/OneginiWrapperErrors.swift [hunk 2]
```diff
@@ -30,7 +33,9 @@ enum WrapperError : Error {
         case .malformedUrl:
             return 8010
         case .noProfileAuthenticated:
-            return 8011
+            return 8012
```
Comment (kfa-onewelcome) [inline ios/OneginiWrapperErrors.swift:36]: where that change comes from?

Comment (MichielVrins) [reply ios/OneginiWrapperErrors.swift:36]: This is to match it to the android error code

**Pull request merged**